tsnet/tsnet.go

```diff
@@ -82,6 +82,13 @@ type Server struct {
 	// used.
 	AuthKey string
 
+	// ControlURL is explicitly not set to signal that
+	// it's not yet configured, which relaxes the CLI "up"
+	// safety net features. It will get set to DefaultControlURL
+	// on first up. Or, if not, DefaultControlURL will be used
+	// later anyway. Can be overridden with TS_CONTROL_URL environment variable.
```

Comment:
This isn't quite the normal doc style we/Go use. Start by saying what it even is.

```
// ControlURL optionally specifies the coordination server URL.
//
// If empty, the TS_CONTROL_URL environment variable is used. If that's empty,
// the Tailscale default is used.
```